nengo_spinnaker/utils/keyspaces.py

```diff
@@ -109,11 +109,9 @@ def __init__(self, routing_tag="routing",
 
         # Add the default keyspace
         nengo_ks = self["nengo"]
-        nengo_ks.add_field("object", tags=[self.routing_tag,
-                                           self.filter_routing_tag])
+        nengo_ks.add_field("object_connection", tags=[self.routing_tag,
```

Comment:
Is this a good name? From my reading of this change, it's become more a global connection ID?